mockbook/src/components/LoginForm.vue

```diff
@@ -0,0 +1,129 @@
+<template>
```

Comment:
Is it possible to make this component generic, such that the same code can be used across this and the doctorapp?